app/views/candidate_interface/content/providers.html.erb

```diff
@@ -6,15 +6,15 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <p class="govuk-body">You can apply to the following training providers and courses using <%= service_name %>. Providers not signed up to <%= service_name %> can only receive applications through UCAS.</p>
-    <h2 class="govuk-heading-s">We suggest you use Apply for teacher training if:</h2>
+    <p class="govuk-body">Not all training providers are signed up to ‘Apply for teacher training’.</p>
```

Comment:
```suggestion
    <p class="govuk-body">Not all training providers are signed up to ‘<%= service_name %>’.</p>
```

Reply:
Hi Paul,

I've now committed this suggested change for review.

Thanks

George